### PR TITLE
[ETP-473]Tests/v2 snapshots part 2

### PR DESCRIPTION
**Pull request by rafsuntaskin**
[ETP-473]

Adding more tests over the previous PR #1787

[ETP-473]: https://moderntribe.atlassian.net/browse/ETP-473

### CODE REVIEW

**Review by juanfra — CHANGES_REQUESTED** [review]
This is great Rafsun! Just left a couple comments.

File: tests/wpunit/Tribe/Tickets/Partials/V2/Tickets/Item/Content/DescriptionTest.php [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+		$driver = $this->get_html_output_driver();
+
+		$driver->setTolerableDifferences( [
+				$args['ticket']->ID,
+			]
+		);
+
+		// Make sure we have the Modal class added
+		$this->assertContains( 'tribe__details__content1', $html );
```
[the remark this answers is not in the document]
Comment (rafsuntaskin) [reply tests/wpunit/Tribe/Tickets/Partials/V2/Tickets/Item/Content/DescriptionTest.php:99]: Yes,  this is intentional. Currently, we have this class like this only for modal: https://github.com/moderntribe/event-tickets/blob/06eb5ae459448565e7bf38d96318a767da012906/src/views/v2/tickets/item/content/description.php#L31

So, I just double-checked this class where it should be present for the `is_modal = true` case.

[the remark this answers is not in the document]
Comment (juanfra) [reply tests/wpunit/Tribe/Tickets/Partials/V2/Tickets/Item/Content/DescriptionTest.php:99]: Shouldn't it be `tribe__details__content__modal` then? If it's concatenating a `1` I guess that the ternary is not doing what it supposed to do.

[the remark this answers is not in the document]
Comment (rafsuntaskin) [reply tests/wpunit/Tribe/Tickets/Partials/V2/Tickets/Item/Content/DescriptionTest.php:99]: Yes, this needs to be fixed in a separate PR as I am not sure if changing the will impact any CSS or not. So, I just went with the snapshots for the current output.
Currently, this is happening in the stable version: http://p.tri.be/rct8Kd

[the remark this answers is not in the document]
Comment (juanfra) [reply tests/wpunit/Tribe/Tickets/Partials/V2/Tickets/Item/Content/DescriptionTest.php:99]: Ok, we should be ticketing this, maybe we can tell Leo and be sure we don't miss this in the future by creating a ticket, referencing these comments.

**Review by juanfra — DISMISSED** [review]
LGTM 😎

**Review by sc0ttkclark — DISMISSED** [review]

File: tests/wpunit/Tribe/Tickets/Partials/V2/Tickets/Item/Content/DescriptionTest.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (sc0ttkclark) [reply tests/wpunit/Tribe/Tickets/Partials/V2/Tickets/Item/Content/DescriptionTest.php:99]: New ticket for this +1

[the remark this answers is not in the document]
Comment (rafsuntaskin) [reply tests/wpunit/Tribe/Tickets/Partials/V2/Tickets/Item/Content/DescriptionTest.php:99]: Created ticket:
https://moderntribe.atlassian.net/browse/ET-920

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**